docker/Dockerfile

```diff
@@ -252,8 +275,68 @@ RUN set -e; \
     rm -rf ~/.cache/pip; \
     truncate -s0 ~/.bash_history;
 
+# TODO(chenhao94) Build packages for arm64.
+#
+# For now we only build packages for amd64 because CI is running on amd64 machines.
+# Otherwise the docker image cross-compiling will be too slow.
+RUN set -e; \
+    export CRIS_IMG_ARCH="$(set -e; \
+        case $(uname -m) in \
+        'amd64' | 'x86_64' | 'x64') \
+            echo 'amd64'; \
+            ;; \
+        'arm64' | 'aarch64') \
+            echo 'arm64'; \
+            ;; \
+        *) \
+            echo "$TARGETARCH"; \
+            ;; \
+        esac)"; \
+    /etc/roaster/scripts/setup.sh \
+        fpm \
+        "$([ "_$CRIS_IMG_ARCH" == '_amd64' ] && echo 'ccache')" \
+        ; \
+    truncate -s0 ~/.bash_history;
+
 RUN set -e; \
-    /etc/roaster/scripts/setup.sh fpm ccache; \
+    export CRIS_IMG_ARCH="$(set -e; \
+        case $(uname -m) in \
+        'amd64' | 'x86_64' | 'x64') \
+            echo 'amd64'; \
+            ;; \
+        'arm64' | 'aarch64') \
+            echo 'arm64'; \
+            ;; \
+        *) \
+            echo "$TARGETARCH"; \
+            ;; \
+        esac)"; \
+    if [ "_$CRIS_IMG_ARCH" == '_arm64' ] ; then \
+        BAZEL_INSTALL_DIR='/usr/local/bin'; \
+        BAZEL_VERSION='5.2.0'; \
+        BAZEL_DEB_PACAKGE_NAME="bazel-$BAZEL_VERSION-arm64.deb"; \
+        for retry in $(seq 10 -1 1); do
+            curl \
+                -fsSL \
+                --retry 10 \
+                --retry-connrefused \
+                -o bazel \
+                "https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-linux-arm64" && break; \
+            rm bazel; \
+            [ "$retry" -gt 1 ] && echo "Retry. $(expr "$retry" - 1) time(s) left." && sleep 5;
+        done; \
+        chmod +x bazel; \
+        fpm \
+            -t deb -s dir \
+            -p "$BAZEL_DEB_PACAKGE_NAME" \
+            --name bazel \
+            --version "$BAZEL_VERSION" \
+            --architecture native \
+            "bazel=$BAZEL_INSTALL_DIR/bazel"; \
+        sudo DEBIAN_FRONTEND=noninteractive apt-get install -y "$(pwd)/$BAZEL_DEB_PACAKGE_NAME"; \
```

Comment:
Wipe the cache after installation.
https://github.com/xkszltl/Roaster/blob/7cf0111b6b12296fe7d7d2736c1cbda0b0d69957/setup.sh#L252-L254